view_post.php

```diff
@@ -0,0 +1,62 @@
+<html>
+
+	<head>
+		<title> View Blog Posts </title>
+		<link rel='stylesheet' href='maindesign.css' />
+	</head>
+
+	<body>
+
+<?php
+
+$sorter= array(
+	'Blog1'=>array(
+		'<h1 class="title">Halfway Done</h1>',
+		'<img src="sophsResized.jpg" alt="All the Amateur sophomores at Senior Send Off." class="photo">',
+		'<p>
+			Every time I listen to the song Dusty Trails by Lucius I get a little bit sad.
+			Theres a part at the end that speaks a lot to where I am right now. It goes: <br> <br>
+			Im halfway to misery <br>
+			Some say when you go halfway theres still plenty of time to return <br>
+			Oh, am I halfway to heaven? <br>
+			Some may say when you go halfway, you only have halfway to go. <br> <br>
+			I cant believe sophomore year is over. I cant believe Ill be a junior soon. That feels so OLD.</p>)'),
+	'Blog2'=>array(
+		'<h1 class="title">Summer in St. Louis</h1>',
```

Comment:
I'd give each of these items in these sub-arrays keys. Like:
```
'Blog2'=>array(
   'Title'=>'Summer in St. Louis',
   'Image'=>'stl.jpg',
   'Body'=>"<p>...</p>"
)
```
This will make it easier to pull that info out later on when you're echoing it (you don't have to remember that 2 = the body. It also makes the code more maintainable. Like, if you want to break the image into  "src" and "alt" fields, you'd mess up the order of whatever comes after that in the array.